app/views/content/blog/the-head-of-science-inspiring-students-in-blackpool.md

```diff
@@ -42,4 +42,4 @@ My favourite part of the job is being in the classroom, without a doubt. There
 
 Rosalind Franklin, who made huge contributions to the understanding of DNA structure yet received no recognition for her work until after she died. Her data was used as the basis of another study by other scientists who went on to receive a Nobel Prize. Also Katherine Johnson, a mathematician who worked on NASA’s space missions. She solved equations by hand for rocket launches and trajectories to get them to launch and land in the right places, and I can’t even begin to imagine how hard that would be!  
 
-Inspired to bring your passion to the classroom? [Sign up for an adviser](/tta-service) and start your journey to becoming a teacher.
+Inspired to bring your passion to the classroom? [Talk to an adviser](/tta-service) with years of teaching experience who can help you find out what life in the classroom is really like.
```

Comment:
Very minor point @EllieNodder but from an accessibility perspective, this means we have two links on this page with different link text going to the same destination (Get an adviser link in the footer). Could this one say 'Get an adviser'? Or point to the TTA page?

But very minor!

Reply:
oo yeah good spot - shall change to the TTA promo page